### PR TITLE
add host_volume for nomad client 

### PR DESCRIPTION
**Pull request by ducminhle**
Fix https://github.com/brianshumate/ansible-nomad/issues/74

### CODE REVIEW

File: tasks/host_volume.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: create volume {{ item['name'] }}
+  file:
+    path: "{{ item['path'] }}"
+    owner: "{{ nomad_user }}"
```
Comment (Legogris) [inline tasks/host_volume.yml]: @ducminhle How about `item['owner'] || nomad_user` (allowing optional setting of ownership)?

File: tasks/host_volume.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    group: "{{ nomad_user }}"
```
Comment (Legogris) [inline tasks/host_volume.yml]: We shouldn't assume group corresponds to member. EIther:
`item['group'] || nomad_group` (see above) or if nothing else `nomad_group`

File: tasks/host_volume.yml (hunk 1 above)
Comment (ducminhle) [reply tasks/host_volume.yml]: Thanks for your suggest.

**Comment by brianshumate** [comment]
Thank you!

**Pull request merged**